src/main/resources/templates/db_scripts/schema.json

```diff
@@ -742,6 +747,11 @@
     }
   ],
   "scripts": [
+    {
+      "run": "before",
+      "snippetPath": "isbn.sql",
```

Comment:
Would it be better to rename the `isbn.sql` file and this line to be a bit more descriptive, such as isbnNormalize.sql (or something similar)?

Reply:
This is the .sql file for the isbn field declared in line 439.
When we also add support for issn and isxn we may rename the file to isbn-issn-isxn.sql.

Reply:
I think it is a good idea to rename the file to describe what it is intended to do.

As I understand it, it defines a function for normalising ISBN values, maybe `create-normalize-isbns-function.sql`?

Reply:
Renamed to create-normalize_isbns-function.sql